bindings/nodejs/lib/wallet/wallet.ts

```diff
@@ -1223,6 +1228,145 @@ export class Wallet {
         );
     }
 
+    /**
+     * Create a delegation.
+     *
+     * @param params The options for creating a delegation.
+     * @param transactionOptions Additional transaction options
+     * or custom inputs.
```

Comment:
Does this really not fit on a line?

Reply:
There is more of it if ever you want to change them all...

Reply:
>:(